### PR TITLE
[12.0] base_report_to_printer: Do no print e-mail attachment

### PR DESCRIPTION
**Pull request by rven**

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Comment by rven** [comment]
@pedrobaeza Travis build fails on "sh: 0: Can't open /etc/init.d/xvfb" how can I fix this?

**Comment by pedrobaeza** [comment]
Please rebase now after 74d1a3f81217bdb0eefd915e54d095d2f1b27281

**Review by jimhoefnagels — APPROVED** [review]

**Comment by rven** [comment]
@pedrobaeza could this be merged?

**Comment by pedrobaeza** [comment]
You want to go very quick, but OK being something very straight-forward: /ocabot merge

File: base_report_to_printer_mail/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    'website': 'https://github.com/OCA/report-print-send',
+    'license': 'AGPL-3',
+    "depends": [
+        'mail',
+        'base_report_to_printer',
+    ],
+    'data': [
+    ],
+    'installable': True,
+    'auto_install': True,
```
Comment (etobella) [inline base_report_to_printer_mail/__manifest__.py:17]: Why this is auto_install?

**Pull request merged**